templates/music/album_detail.html

```diff
@@ -12,58 +13,142 @@
+            <a href="{{ marketplace_url.url }}" target="_blank">
+                {% with "img/marketplace/"|add:marketplace_url.medium|add:".svg" as medium_image %}
+                    <img class="marketplace-icons" src="{% static medium_image %}" alt="{{ marketplace_url.get_medium_display }}" />
+                {% endwith %}
+            </a>
+        {% endfor %}
+    </div>
+    {% if album.release_date %}
+        <h5 class="padding">Release date: {{ album.release_date }}</h5>
+    {% endif %}
+    <div class="break-small">
+        <hr>
+    </div>
+    <div>
     {% if user_is_investor %}
-        <h4>FOR INVESTORS</h4>
-        <div class="break-small">
-            <hr>
-        </div>
         <div class="download-container">
             <a class="download-button" href="{{ album.audio.get_temporary_url }}" download>
                 Download
                 <i class="fa fa-download" aria-hidden="true"></i>
             </a>
         </div>
+        <div class="break-small">
+            <hr>
+        </div>
+    {% endif %}
+
+        <h4>REVENUE</h4>
+
+            <div class="bar-chart">
+                <canvas id="line-canvas" height="650px" width="1000px"></canvas>
+            </div>
+        </div>
+
+
+    <script>
+        var randomScalingFactor = function(){ return Math.round(Math.random()*100)};
+        var lineChartData = {
+            labels : ["January","February","March","April","May","June","July"],
+            datasets : [
+                {
+                    label: "Artist Revenue",
+                    fillColor : "rgba(220,220,220,0.2)",
+                    strokeColor : "rgba(220,220,220,1)",
+                    pointColor : "rgba(220,220,220,1)",
+                    pointStrokeColor : "#fff",
+                    pointHighlightFill : "#fff",
+                    pointHighlightStroke : "rgba(220,220,220,1)",
+                    data : [randomScalingFactor(),randomScalingFactor(),randomScalingFactor(),randomScalingFactor(),randomScalingFactor(),randomScalingFactor(),randomScalingFactor()]
+                },
+                {
+                    label: "Investor income",
+                    fillColor : "rgba(151,187,205,0.2)",
+                    strokeColor : "rgba(151,187,205,1)",
+                    pointColor : "rgba(151,187,205,1)",
+                    pointStrokeColor : "#fff",
+                    pointHighlightFill : "#fff",
+                    pointHighlightStroke : "rgba(151,187,205,1)",
+                    data : [randomScalingFactor(),randomScalingFactor(),randomScalingFactor(),randomScalingFactor(),randomScalingFactor(),randomScalingFactor(),randomScalingFactor()]
+                }
+            ]
+
+        }
+
+
+    </script>
+        <br/>
+        <br/>
+        <h4>Streams / Downloads</h4>
+{% for disc in album.discs %}
+    {% for track in disc %}
+    <head>
+        <title>Bar Chart</title>
+        <script src="../Chart.js"></script>
+    </head>
+    <body>
```

Comment:
This part of the HTML is already in the `<body>` tag, so you don't need the body tag defined here.